modules/standard/Time.chpl

```diff
@@ -1052,146 +1058,160 @@ enum Day       { sunday=0, monday, tuesday, wednesday, thursday, friday, saturda
                              microseconds=timeSinceEpoch(1));
     return unixEpoch + td;
   }
 
-  /* The `datetime` that is `timestamp` seconds from the epoch */
-  proc type datetime.fromTimestamp(timestamp: real) {
-    return datetime.fromTimestamp(timestamp, nil);
+  /* The `dateTime` that is `timestamp` seconds from the epoch */
+  proc type dateTime.fromTimestamp(timestamp: real) {
+    return dateTime.fromTimestamp(timestamp, nil);
   }
 
-  /* The `datetime` that is `timestamp` seconds from the epoch */
+  /* The `dateTime` that is `timestamp` seconds from the epoch */
   @unstable("tz is unstable; its type may change in the future")
-  proc type datetime.fromTimestamp(timestamp: real,
+  proc type dateTime.fromTimestamp(timestamp: real,
                                    in tz: shared Timezone?) {
     if tz.borrow() == nil {
       var t = (timestamp: int, ((timestamp - timestamp: int)*1000000): int);
       const lt = getLocalTime(t);
-      return new datetime(year=lt.tm_year+1900, month=lt.tm_mon+1,
+      return new dateTime(year=lt.tm_year+1900, month=lt.tm_mon+1,
                           day=lt.tm_mday,       hour=lt.tm_hour,
                           minute=lt.tm_min,     second=lt.tm_sec,
                           microsecond=t(1));
     } else {
-      var dt = datetime.utcFromTimestamp(timestamp);
+      var dt = dateTime.utcFromTimestamp(timestamp);
       return (dt + tz!.utcOffset(dt)).replace(tz=tz);
     }
   }
 
-  /* The `datetime` that is `timestamp` seconds from the epoch in UTC */
-  proc type datetime.utcFromTimestamp(timestamp) {
-    return unixEpoch + new timedelta(seconds=timestamp: int, microseconds=((timestamp-timestamp: int)*1000000): int);
+  /* The `dateTime` that is `timestamp` seconds from the epoch in UTC */
+  proc type dateTime.utcFromTimestamp(timestamp) {
+    return unixEpoch + new timeDelta(seconds=timestamp: int, microseconds=((timestamp-timestamp: int)*1000000): int);
   }
 
-  /* The `datetime` that is `ordinal` days from 1-1-0001 */
-  proc type datetime.fromOrdinal(ordinal) {
-    return datetime.combine(date.fromOrdinal(ordinal), new time());
+  /* The `dateTime` that is `ordinal` days from 1-1-0001 */
+  proc type dateTime.fromOrdinal(ordinal) {
+    return dateTime.combine(date.fromOrdinal(ordinal), new time());
   }
 
-  /* Form a `datetime` value from a given `date` and `time` */
-  proc type datetime.combine(d: date, t: time) {
-    return new datetime(d.year, d.month, d.day,
+  /* Form a `dateTime` value from a given `date` and `time` */
+  proc type dateTime.combine(d: date, t: time) {
+    return new dateTime(d.year, d.month, d.day,
                         t.hour, t.minute, t.second, t.microsecond, t.timezone);
   }
 
-  /* Methods on datetime values */
+  /* Methods on dateTime values */
 
-  /* Get the `date` portion of the `datetime` value */
-  proc datetime.getdate() {
+  @deprecated(notes="'dateTime.getdate' is deprecated. Please use 'dateTime.getDate' instead")
```

Comment:
Same here. can this link to the new method in the generated docs? ``:proc:`dateTime.getDate` ``?